src/HomogeneousCalorimeter_geo.cpp

```diff
@@ -58,6 +40,63 @@ template <class XmlComp> Position get_xml_xyz(XmlComp& comp, dd4hep::xml::Strng_
   return pos;
 }
 
+static Volume inner_support_collar(Detector& desc, xml_comp_t handle) {
+  // This consists of two circular tubes joined by straight sections
+
+  Material inner_ring_material = desc.material(handle.materialStr());
+
+  double electron_rmin         = handle.attr<double>(_Unicode(electron_rmin));
+  double electron_rmax         = handle.attr<double>(_Unicode(electron_rmax));
+  double proton_rmin           = handle.attr<double>(_Unicode(proton_rmin));
+  double proton_rmax           = handle.attr<double>(_Unicode(proton_rmax));
+  double straight_section_tilt = handle.attr<double>(_Unicode(straight_section_tilt));
```

Comment:
Is the tilt of the section the basic parameter here? Should this not be the separation of the centers, from which the tilt is derived?